catalog.html

```diff
@@ -10,7 +10,8 @@
     <header class="main-header-general">
       <nav class="navigation">
         <a class="navigation-logo" href="index.html">
-          <img class="page-header-logo" src="" alt="Логотип тургорода Седона">
+          <img class="page-header-logo" src="images/general-images/logo.png" weight="140" height="70"
```

Comment:
```suggestion
          <img class="page-header-logo" src="images/general-images/logo.png" width="140" height="70"
```
Такая ошибка во всех `img`. Такого атрибута как `weight` нет, есть `width`.